dataland-frontend/src/components/general/TheGetInTouch.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <section
+    v-if="getInTouchSection && getInTouchSection.cards"
+    class="getintouch"
+    role="region"
+    aria-labelledby="getintouch-heading"
+  >
+    <div class="getintouch__wrapper">
+      <h2 id="getintouch-heading" class="getintouch__title">
+        {{ getInTouchSection.text[0] }}
+        <span v-if="getInTouchSection.text[1]"> {{ getInTouchSection.text[1] }}</span>
+      </h2>
+      <div v-if="getInTouchSection.cards[1]" class="getintouch__image" role="img" aria-label="Contact Image">
+        <img :src="getInTouchSection.cards[1].icon" :alt="getInTouchSection.cards[1].title" />
+      </div>
+      <div class="getintouch__text" grid-column="15 / 17">
+        <p class="getintouch__text-title">{{ getInTouchSection.cards[1].title }}</p>
+        <p class="getintouch__text-text">{{ getInTouchSection.cards[1].text }}</p>
+        <button class="getintouch__text-button" @click="openEmailClient">{{ getInTouchSection.text[0] }}</button>
+      </div>
+    </div>
+  </section>
+</template>
+
+<script setup lang="ts">
+import { computed } from "vue";
+import type { Section } from "@/types/ContentTypes";
+
+const { sections } = defineProps<{ sections?: Section[] }>();
+
+const getInTouchSection = computed(() => {
+  return sections?.find((section) => section.title === "Get in touch") || null;
+});
+
+const openEmailClient = (): void => {
+  if (getInTouchSection.value && getInTouchSection.value.cards) {
+    const cards = getInTouchSection.value.cards;
+
+    const email = cards[2]?.icon || "";
+    const subject = cards[2]?.title || "";
+    const body = cards[2]?.text || "";
+
+    if (email && subject && body) {
+      const mailtoString = `mailto:${email}?subject=${encodeURIComponent(subject)}&body=${encodeURIComponent(body)}`;
+      window.location.href = mailtoString;
+    }
```

Comment:
This seems like duplicate code when you compare it to the component "EmailButton.vue".

Could we somehow reuse the "EmailButton.vue" component here?

Reply:
Yes, I will create a composable or component for this, but a form-to-mail will take its place on the next version, so it's temporary

Reply:
Ok